quixstreams/state/rocksdb/windowed/transaction.py

```diff
@@ -51,15 +55,18 @@ def as_state(self, prefix: Any = DEFAULT_PREFIX) -> WindowedTransactionState:
             ),
         )
 
-    def get_latest_timestamp(self) -> int:
-        return self._latest_timestamp_ms
+    def get_latest_timestamp(self, prefix: bytes) -> int:
+        cached_ts = self._latest_timestamps.get(prefix)
+        if cached_ts is not None:
+            return cached_ts
 
-    def _validate_duration(self, start_ms: int, end_ms: int):
-        if end_ms <= start_ms:
-            raise ValueError(
-                f"Invalid window duration: window end {end_ms} is smaller or equal "
-                f"than window start {start_ms}"
-            )
+        stored_ts = self.get(
+            key=LATEST_TIMESTAMP_KEY,
+            prefix=prefix,
+            cf_name=LATEST_TIMESTAMPS_CF_NAME,
+        )
+        self._latest_timestamps[prefix] = stored_ts
+        return stored_ts or 0
```

Comment:
I think we can do `self.get(..., default=0)` here and remove `... or 0`. This way we will end up with 0 in the `_latest_timestamps` cache which is better than useless `None`.

Reply:
Agree, updated 👍